implementations/rust/ockam/ockam_api/src/logs/logging_configuration.rs

```diff
@@ -300,57 +269,145 @@ pub fn logging_enabled() -> ockam_core::Result<LoggingEnabled> {
     }
 }
 
-/// Return the log level, using the legacy environment variable if defined,
-/// or the current one.
-fn log_level(preferred_log_level: Option<Level>) -> ockam_core::Result<Level> {
-    let is_legacy_variable_set = is_set::<LevelVar>(OCKAM_LOG)?;
-    let env_variable = if is_legacy_variable_set {
-        println!("The OCKAM_LOG variable is deprecated. Please use: OCKAM_LOGGING=true OCKAM_LOG_LEVEL=trace (with the desired level) instead.");
-        OCKAM_LOG
-    } else {
-        OCKAM_LOG_LEVEL
-    };
-    get_log_level(env_variable, preferred_log_level)
+/// Return the strategy to use for reporting logging/tracing errors
+pub fn global_error_handler() -> ockam_core::Result<GlobalErrorHandler> {
+    match get_env::<GlobalErrorHandler>(OCKAM_TRACING_GLOBAL_ERROR_HANDLER)? {
+        Some(v) => Ok(v),
+        None => Ok(GlobalErrorHandler::LogFile),
+    }
 }
 
-/// A user can pass a preferred logging level via a -vvv argument (with the number of 'v' indicating
-/// the log level). If that argument is not present, we get the log level from an environment variable.
-fn get_log_level(
-    variable_name: &str,
-    preferred_log_level: Option<Level>,
-) -> ockam_core::Result<Level> {
-    match preferred_log_level {
-        Some(level) => Ok(level),
-        None => get_env_with_default(
-            variable_name,
+/// Represents the relationship between a log level and the crates that will
+/// get monitored at that level
+pub struct LogLevelWithCratesFilter {
+    pub level: Level,
+    pub crates_filter: CratesFilter,
+    /// Indicates if the log level was explicitly set by the commands' `verbose` flag
+    pub explicit_verbose_flag: bool,
+}
+
+impl LogLevelWithCratesFilter {
+    /// Create a new LogLevel using one of the environment variables
+    /// (the legacy OCKAM_LOG variable or the current OCKAM_LOG_LEVEL variable).
```

Comment:
Maybe we can drop the `OCKAM_LOG` variable. It's been a while now.

Reply:
Let's do it!